server.rb

```diff
@@ -32,6 +32,7 @@ class Server < ::Sinatra::Base
         end
 
         FileUtils.mkdir_p settings.cache_dir
+        Dir.glob("#{settings.cache_dir}/*.json").each { |file| File.delete(file)}
```

Comment:
Yeah, the version idea sounds like a good one. I'll see how it goes

Reply:
hmm I'm thinking (this might be my laziness talking) but if we do the versioning, we might not even have to delete any files here. This'll make for smoother rollbacks. 

If you're short on disk space from this, you could probably make a cron job to delete files that have not been accessed in a long time and files that have been created a long time ago (so you'll allow them to be refreshed in the case that a station might be created).

idk, what do you think?

Reply:
Ahh I see, instead of cleaning up, just hardcode the version in the code and put it in the filenames.  Yeah, I mean, this isn't software everybody is gonna go install somewhere -- disk is cheap where I live, so screwit.  Simple is best.  We can manually cleanup if/when this becomes an issue.  This isn't gonna happen again until it gets extended for some other region's data anyway..

Agree.